tests/feature/test_background.py

```diff
@@ -2,16 +2,16 @@
 
 import textwrap
 
-FEATURE = '''\
+FEATURE = """\
```

Comment:
why this change? I prefer the previous version where we didn't need to `\` escape chars inside this heredoc

Reply:
Sorry, it was my overzealous replacing from fixing another test.  This was unintentional.  Teaches me to be careful with find and replace!